apm-agent-core/src/main/java/co/elastic/apm/impl/ElasticApmTracer.java

```diff
@@ -119,10 +119,6 @@ public void onChange(ConfigurationOption<?> configurationOption, Double oldValue
         }
     }
 
-    public static ElasticApmTracerBuilder builder() {
-        return new ElasticApmTracerBuilder();
```

Comment:
Why are you moving away from having a singleton?

Reply:
I just inlined the call to new ElasticApmTracerBuilder() and removed the static method on `ElasticApmTracer` here.

Reply:
Nevermind I may have missread the method :facepalm: I think I was thinking you always returned a single instance.

Reply:
Actually, I previously did some changes around that. There is no singleton instance of `ElaticApmTracer` anymore. It gets instantiated in `premain` and all `ElasticApmInstrumentation` get the instance via a `init` method. This makes Unit testing with different settings and different instances of `ElaticApmTracer` way easier.